setup.py

```diff
@@ -10,16 +10,18 @@
 
 if __name__ == "__main__":
     setup(name="lithium",
-          version="0.1",
+          version="0.2.0",
           entry_points={
-              "console_scripts": ["lithium = lithium:main"]
+              "console_scripts": ["lithium = lithium.reducer:main"]
           },
-          packages=["lithium", "lithium.lithium"],
-          package_data={"": [
-              "interestingness/*",
-              "lithium/doc/*",
-              "lithium/examples/*.*",
-              "lithium/examples/arithmetic/*"
+          packages=[
+              "lithium",
+              "lithium.interestingness",
+          ],
+          package_data={"lithium": [
+              "docs/*",
+              "docs/examples/*",
```

Comment:
These 2 lines (`docs/*` and `docs/examples/*`) will require `*` -> `*.*`. I'm not sure why this didn't come up in local testing, but when I tried installing in EC2 with pip I got this:
```
Installing collected packages: lithium
  Running setup.py install for lithium: started
    Running setup.py install for lithium: finished with status 'error'
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-Vu8_MK-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install
--record /tmp/pip-9hH3Iv-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/lithium
    copying src/lithium/reducer.py -> build/lib.linux-x86_64-2.7/lithium
    copying src/lithium/__init__.py -> build/lib.linux-x86_64-2.7/lithium
    copying src/lithium/__main__.py -> build/lib.linux-x86_64-2.7/lithium
    creating build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/utils.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/__init__.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/range.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/outputs.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/crashes.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/diff_test.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/timed_run.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    copying src/lithium/interestingness/hangs.py -> build/lib.linux-x86_64-2.7/lithium/interestingness
    creating build/lib.linux-x86_64-2.7/lithium/docs
    copying src/lithium/docs/algorithm.md -> build/lib.linux-x86_64-2.7/lithium/docs
    copying src/lithium/docs/related.txt -> build/lib.linux-x86_64-2.7/lithium/docs
    error: can't copy 'src/lithium/docs/examples': doesn't exist or not a regular file

    ----------------------------------------
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-Vu8_MK-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-9hH3Iv
-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-Vu8_MK-build/
```

I know I'd seen that before which is why I had `*.*` in the original, but again I don't know why this didn't happen in my local testing.